### DISCUSSION TITLE
弹出选择数据源的底部弹窗时，直接全屏展开会不会更好

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#163] [Ideas]
收起状态下其实看不到太多信息，大部分情况都要上滑展开到全屏的，所以是不是可以直接改成默认全屏

值得一提的是现在缓存页面的同样的弹窗是默认全屏的，我感觉体验还不错

**Comment by NeKoOuO** [comment]
> 收起状态下其实看不到太多信息，大部分情况都要上滑展开到全屏的，所以是不是可以直接改成默认全屏
> 
> 值得一提的是现在缓存页面的同样的弹窗是默认全屏的，我感觉体验还不错

根據我的使用這個是確實的

我覺得可以加一個設計就是當你在前集設定好了默認觀看路徑的時候

如果搜索到的資源只有一個的時候就直接默認加載

因為我每次在點的時候 會遇到一個問題就是 當我選擇資源的時候 要等他把全部跑完 才有辦法選擇去點 然後其實這樣那個默認設置就會沒什麼屁用 因為還是要自己再去點點點

**1.要把資源來源選擇放在最高優先級**

目前我遇到的問題就是 當我點進去資源選擇的時候 最下面的來源可能我的默認設置是花園 但是系統都會幫我選蜜柑 然後目前的設計又不會說選擇蜜柑它就給你加載蜜柑而已 只是加個過濾器而已 所以說還是要等他跑完 

然後我就會遇到我要拉到最下面 點擊另一個來源來切換過濾器 但是由於加載還是持續進行中的 所有就會導致我一直點不到 因為會有字幕組一直彈出來往下擠 然後如果點錯的話又會需要在從新加載一次 

一般來說字幕組都是資源選擇的最後階段的篩選 所以說放在最後面真的不太符合邏輯
